purchase_order_line_schedule/views/purchase_order_views.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <field name="date_approve" position="after">
+                <field name="has_qty_to_schedule" invisible="True" />
+            </field>
+            <xpath
+                expr="//notebook//field[@name='order_line']/tree/field[@name='date_planned']"
+                position="after"
+            >
+                <field name="schedule_line_ids" invisible="1" />
+                <button
+                    string="Delivery Schedule"
+                    name="action_open_schedule_lines"
+                    type="object"
+                    icon="fa-list"
+                    optional="hide"
+                />
+                <field
+                    name="qty_to_schedule"
+                    attrs="{'column_invisible': [('parent.has_qty_to_schedule', '=', False)]}"
+                />
```

Comment:
Would be nice to always see the schedule not only when there is something to schedule.